httprequest_lego_provider/forms.py

```diff
@@ -62,6 +62,17 @@ def validate(self, value):
                 message="Please provide a valid FQDN", code="invalid", params={"value": value}
             )
 
+    def to_python(self, value) -> str:
```

Comment:
I'm wondering if i'm missing where this function is being used?

Reply:
It is called by the framework. See https://docs.djangoproject.com/en/5.0/howto/custom-model-fields/